### DISCUSSION TITLE
PDS per collectible or many collectibles per PDS?

### DISCUSSION THREAD

**Discussion by latenssi** [flow-hydraulics/flow-pds#4] [General]
Do we want to run a PDS per collectible type or one PDS to handle any number of different collectibles?

**Comment by aphelionz** [comment]
I assume you mean one PDS for "My Medium-Sized Pony" and one for "The National Sportsball League" etc. What are the implications on the go code itself for one-PDS-per vs a generic PDS for anything?

**Comment by EricLin2004** [comment]
The idea is we could use one generic PDS for multiple experiences but I agree we should discuss what the implications are on the codebase itself and if it would increase the time to delivery for a first version.

**Comment by latenssi** [comment]
Additionally, can a Pack have NFTs of varying types? I am basing this question on this line in the description document:

> Provide the type and ID numbers (as lists or ranges) of the NFTs to be included in each Bucket.

**Comment by aphelionz** [comment]
I believe this is answered here: 

> NOTE: This design allows anyone to create Packs for any Collectibles. I.e. A third party could create packs containing NBA Top Shots. However, if they did so, it would be a distinct Pack NFT Resource, and there is no possibility of on- or off-chain code accidentally mistaking an official Pack for a third-party Pack. (Provided the NBA Top Shot team properly secures access to the Pack creation Capability of their Pack NFT implementation.)


So, it should be compatible, but you should never be able to _actually_ do it
